Automattic/jetpack
# Synced Order Data: Adds filtering for filtered order data

This PR adds a filter to the filtered order data that is sent to sync.

## Proposed changes:

* In this PR, I've added a filter before returning the `filtered_order_data`, which is sent to sync. This allows plugins that use the Jetpack sync to send additional data to the sync.

### Other information:


## Jetpack product discussion


pfKYZu-Ti-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* In your `functions.php` add this custom PHP:
```php
add_filter( 'jetpack_sync_filtered_hpos_order_data, 'test_filtered_order_data', 10, 2 );

function test_filtered_order_data( $filtered_order_data, $order_object ) {
    l( $filtered_order_data);
    return $filtered_order_data;
}
```

* Perform a order sync operation.
* Navigate to the `debug.log` file and ensure the filtered order data is logged through the filter.
